### PR TITLE
feat(crons): Render timeline view on crons issue details page

### PR DESCRIPTION
**Pull request by davidenwang**
Renders a timeline view in the issue details page like so:

image

image

### CODE REVIEW

File: static/app/components/events/interfaces/crons/cronTimelineSection.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+      },
+    ],
+    {
+      staleTime: 0,
+      enabled: !!monitorSlug && timelineWidth > 0,
+    }
+  );
+
+  if (!monitorSlug) {
+    return null;
```
Comment (evanpurkhiser) [inline static/app/components/events/interfaces/crons/cronTimelineSection.tsx:67]: Do we want to show a loading indicator w/ what @vuluongj20 built?

File: static/app/components/events/interfaces/crons/cronTimelineSection.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+const EventLineLabel = styled(Overlay)<{left: number; timelineWidth: number}>`
+  width: max-content;
+  padding: ${space(0.75)} ${space(1)};
+  color: ${p => p.theme.textColor};
+  font-size: ${p => p.theme.fontSizeSmall};
+  position: absolute;
+  bottom: ${space(1)};
+  left: clamp(0px, ${p => p.left}px, calc(${p => p.timelineWidth}px - 50px));
+  transform: translateX(-50%);
+`;
```
Comment (evanpurkhiser) [inline static/app/components/events/interfaces/crons/cronTimelineSection.tsx:161]: Can you use the `Overlay` component for this?

**Review by evanpurkhiser — APPROVED** [review]
Looking good

**Comment by vuluongj20** [comment]
Let's move the segmented control to the top right corner, vertically centered with the section title, like here:
image

File: static/app/components/events/interfaces/crons/cronTimelineSection.tsx (hunk 1 above)
Comment (davidenwang) [reply static/app/components/events/interfaces/crons/cronTimelineSection.tsx:67]: oh yes definitely let me get that

**Pull request merged**